src/data/sym/basic.lean

```diff
@@ -223,7 +255,7 @@ def map {α β : Type*} {n : ℕ} (f : α → β) (x : sym α n) : sym β n :=
 @[simp] lemma mem_map {α β : Type*} {n : ℕ} {f : α → β} {b : β} {l : sym α n} :
   b ∈ sym.map f l ↔ ∃ a, a ∈ l ∧ f a = b := multiset.mem_map
 
-@[simp] lemma map_id {α : Type*} {n : ℕ} (s : sym α n) : sym.map id s = s :=
+@[simp] lemma map_id {α : Type*} {n : ℕ} (s : sym α n) : sym.map (λ (x : α), x) s = s :=
```

Comment:
Rather than changing this lemma, probably you should just create a `map_id'` version as well.

Reply:
I've just reverted this to its original form since (as far as I can see) there's no use of this modified version.  I guess Khahn may have changed this in response to the linter complaining that this simp lemma isn't in simp-normal form (since `id` can be simplified to `(λ (x : α), x)`).  If it passes CI then I'll leave it as it was.

Reply:
I've added both versions, making the one in terms of `λ` a `simp` lemma. I've also added the following comment so we can find this and remove the duplication if and when the underlying issue is resolved:
```
/-- The duplication in the following two lemmas is to avoid a complaint from the simp_nf linter -/
```